### ISSUE TITLE
Missing: multi_map_server/MultiOccupancyGrid.h during building --> found in https://github.com/HKUST-Aerial-Robotics/Fast-Planner/tree/master/uav_simulator/Utils/multi_map_server

### ISSUE THREAD

**Issue by Twenkid** [ZJU-FAST-Lab/ego-planner-swarm#65]
Hi, you've created an amazing code base both in this project and your lab! Good job!
I tried to compile the ego-planer-swarm project, but either ways for building it fails with the same error, a header is missing and I can't find it in the folders either.
```
...
catkin_make -j1
...
catkin_make -DCMAKE_BUILD_TYPE=Release -j1
```

![image](https://github.com/ZJU-FAST-Lab/ego-planner-swarm/assets/23367640/15813684-28fa-4557-a2be-08a68e4e812e)

I find only Python files with that name and underscore, in multi_map_server.zip  etc.

_MultiOccupancyGrid.py

I found the files here: 

https://github.com/HKUST-Aerial-Robotics/Fast-Planner/tree/master/uav_simulator/Utils/multi_map_server

...and I copied them to the directory, the building continued and succeeded.

![image](https://github.com/ZJU-FAST-Lab/ego-planner-swarm/assets/23367640/c18ca4c3-1460-44fb-b4a8-8e92d61f4f82)

Regards

**Comment by HauserDong** [comment]
Yes, you can also find the requested head files in the following link: https://github.com/HKUST-Aerial-Robotics/plan_utils/tree/master/multi_map_server/msg_gen/cpp/include/multi_map_server

**Comment by lbboier** [comment]
Using "catkin_make"  or "ctrl +shift +B" instend of "catkin_make -j1"

**Comment by zoythum** [comment]
@Twenkid what files did you copy? And where did you copy them? Thanks!

**Comment by Twenkid** [comment]
Hi, see the image with the error message and the links in the messages above. .   .../Utils/multi_map_server ... All 4 header files, if they are missing.
